### ISSUE TITLE
{ Link } not being rendered (2.4.4)

### ISSUE THREAD

**Issue by AdamJo** [developit/preact-router#172]
```
  Windows 10
  Preact: 8.1.0
  Preact-Router: 2.4.4
  Webpack 2.3.3
```
Edit: This works on preact-router 2.4.3, but not 2.4.4.

The link tags are not getting rendered in the webpage. No errors on build.

```
<nav>
  <undefined href="/">Home</undefined>
  <undefined href="/profile">Me</undefined>
  <undefined href="/profile/john">John</undefined>
</nav>
```

I am getting this error on my 8.0 branch of [the Preact-Boilerplate found here](https://github.com/AdamJo/preact-boilerplate/tree/preact8).

When running tests I get the following error for routing.
`undefined is not a constructor (evaluating '(0, _preactRouter.route)('/profile')')`

**Comment by AdamJo** [comment]
Fixed in 2.4.5. Closing the issue.

**Comment by developit** [comment]
Yup, was a bundling error, apologies for that!
